### PR TITLE
Add guide for setting up and using Mapviz in ROS2

### PR DESCRIPTION
**Pull request by shrijitsingh99**
Shows how to setup Mapviz in ROS2. Includes some helper scripts and their explanation that makes integration with robots easier.

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -273,6 +273,8 @@ wiki:
         url: /wiki/tools/stream-rviz/
       - title: Web-Based Visualization using ROS JavaScript Library
         url: /wiki/tools/roslibjs/
+      - title: MapViz for Map Based Visualization in ROS2
+         url: /wiki/planning/mapviz/
```
Comment (smbryan) [inline _data/navigation.yml:277]: Whitespace. `url:` should be aligned with title. Some YAML parsers will throw an error if not. 

You're using spaces, right?

CC: @nevalsar

File: wiki/tools/mapviz.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
+# Mapviz for Map Based Visualization in ROS2
```
Comment (smbryan) [inline wiki/tools/mapviz.md:1]: Missing page meta information. Please see the template.

Also, the first paragraph shouldn't have a header (i.e. "# Mapviz...") or any markdown. Jekyll uses the first paragraph (or the first 100 words) to generate a blurb for your entry. Unfortunately, it pulls what it finds verbatim.

CC: @nevalsar

**Review by smbryan — CHANGES_REQUESTED** [review]
Great start. Needs some minor housekeeping before we can merge.

**Review by nevalsar — CHANGES_REQUESTED** [review]
(See earlier review comments)

**Comment by nevalsar** [comment]
Contributor unresponsive - will raise these changes as a separate MR fixing issues.

**Pull request closed without merging**